### PR TITLE
#157574129 Build Create Requests page

### PR DESCRIPTION
**Pull request by OKiMaureen**
What does this PR do?
It shows a page where users can create requests

Description of Task to be completed?
-users create requests for repairs or maintenance on this page

What are the relevant pivotal tracker stories?
#157574129
Questions:
none

Screenshots:
desktop view
image
mobile view
image
updated image: destop view
image
mobile view
image

Resedesigned images:desktop
image

mobile:
image

### CODE REVIEW

**Comment by paradisekelechi** [comment]
@OKiMaureen 
Can you come up with a better design for this create request page? It looks exactly like the signup page.

File: ui/allrequest.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+!DOCTYPE html>
```
Comment (paradisekelechi) [inline ui/allrequest.html:1]: Is this file necessary for this pull request? @OKiMaureen

Comment (OKiMaureen) [reply ui/allrequest.html:1]: no, It was a mistake
